### PR TITLE
Replaced CITATION.bib by CITATION.cff

### LINKED ISSUE #86: Replace CITATION.bib by CITATION.cff
https://citation-file-format.github.io/cff-initializer-javascript/#/start

### PR DESCRIPTION
**Pull request by Jay-sanjay**
This PR address #86 

closes #86

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - family-names: "Kenens"
+    given-names: "A."
+  - family-names: "Orban"
+    given-names: "D."
+  - name: "contributors"
+title: "BundleAdjustmentModels.jl: Julia repository of bundle adjustment problems"
+month: "July"
+year: 2021
+repository-code: "https://github.com/JuliaSmoothOptimizers/BundleAdjustmentModels.jl"
+doi: "10.5281/zenodo.5167631"
```
Comment (tmigot) [inline CITATION.cff:13]:
```suggestion
keywords:
  - Least squares
  - Julia
  - Bundle Adjustment
identifiers:
  - description: Zenodo archive
    type: doi
    value: 10.5281/zenodo.5167631
```

**Review by tmigot — CHANGES_REQUESTED** [review]
Thanks @Jay-sanjay for the suggestion, hère are some comments

**Comment by Jay-sanjay** [comment]
Thank you for the suggestions sir.

**Comment by tmigot** [comment]
Could you also update the links in the readme.md and https://github.com/Jay-sanjay/BundleAdjustmentModels.jl/blob/cff/docs/src/index.md to point to CFF and not BIB?

**Comment by tmigot** [comment]
@software{Kenens_BundleAdjustmentModels_jl_Julia_repository_2023,
author = {Kenens, Antonin and Orban, Dominique and {contributors}},
license = {MPL-2.0},
month = mar,
title = {{BundleAdjustmentModels.jl: Julia repository of bundle adjustment problems}},
url = {https://github.com/JuliaSmoothOptimizers/BundleAdjustmentModels.jl},
version = {0.3.2},
year = {2023}
}

**Review by tmigot — APPROVED** [review]
Thank you!

**Pull request merged**